app/policies/subscription_policy.rb

```diff
@@ -0,0 +1,15 @@
+class SubscriptionPolicy < ApplicationPolicy
+  def create?
+    signed_in? && !subscribed?
```

Comment:
I think we should remove this policy because policies should check users rights to perform an action. 
In my opinion, we shouldn't check here if the user is already subscribed to another user because it is a uniqueness validation. I think it will be better to add validation to the subscriptions model to check that `follower_id` and `followee_id` are unique.
In case, that user will try to create the second subscription, the validation will fail.
What do you think about it?